packages/web-player-component/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Web Player Component Demo</title>
+    </head>
+    <body>
+        <!-- Browsers will block autoplay if muted is false -->
+        <player-component source="https://maitv-vod.lab.eyevinn.technology/VINN.mp4/master.m3u8" starttime="25" muted autoplay></player-component>
```

Comment:
I propose we add the eyevinn name in the component, suggesting:
`<eyevinn-video>`